Core/Core/Api/GraphQL/Models.cs

```diff
@@ -396,6 +396,23 @@ public class ServerInfo
   /// see <see cref="Speckle.Core.Credentials.AccountManager"/>
   /// </remarks>
   public string url { get; set; }
+
+  public ServerMigration migration { get; set; }
+}
+
+public class ServerMigration
+{
+  /// <summary>
+  /// One of the base fields for our account upgrade mechanism.
+  /// Shows the url that this account now lives at after upgrading to FE2
+  /// </summary>
+  public Uri movedTo { get; set; }
+
+  /// <summary>
+  /// One of the base fields for our account upgrade mechanism.
+  /// Shows the url that this account used to live at before being upgraded to FE2
+  /// </summary>
+  public Uri movedFrom { get; set; }
```

Comment:
This docs are deceiving. `ServerMigration` class knows nothing of any Account, it's purely server metadata.

`movedFrom` and `movedTo` show the "new URL where this server is now deployed" and "the old RUL where this server used to be deployed" respectively.

Reply:
You're right this is misleading